### PR TITLE
#121 Added support for configured databasechangelog/lock table names.

### PR DESCRIPTION
**Pull request by cbuschka**
This change adds support for customization of changelog and changeloglock table names. (see issue #121 )

### CODE REVIEW

**Comment by mescanne** [comment]
This is fantastic -- I have had a look through and it looks really solid. I'm going to discuss with Knut (olavloite) next week and we'll get it merged.

Thanks

File: src/test/java/liquibase/ext/spanner/sqlgenerator/CreateDatabaseChangeLogLockTableGeneratorSpannerTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package liquibase.ext.spanner.sqlgenerator;
```
Comment (olavloite) [inline src/test/java/liquibase/ext/spanner/sqlgenerator/CreateDatabaseChangeLogLockTableGeneratorSpannerTest.java:1]: nit: would you mind adding (just copy-paste) a copyright header to this file and the other test file?

**Review by olavloite — COMMENTED** [review]
Thanks for adding this! Looks good to me (with a tiny nit on missing copyright headers)

File: src/test/java/liquibase/ext/spanner/sqlgenerator/CreateDatabaseChangeLogLockTableGeneratorSpannerTest.java (hunk 1 above)
Comment (cbuschka) [reply src/test/java/liquibase/ext/spanner/sqlgenerator/CreateDatabaseChangeLogLockTableGeneratorSpannerTest.java:1]: Sure, sorry for missing it.

**Review by olavloite — APPROVED** [review]

**Comment by cbuschka** [comment]
♥️

**Pull request merged**